packit/utils.py

```diff
@@ -250,3 +254,39 @@ def get_namespace(url: str) -> str:
         return namespace
     except IndexError:
         return None
+
+
+def get_wildcard_resolved_sync_files(list_to_sync: List) -> List:
```

Comment:
please utilize the config class here, not a list of dicts

Reply:
Import `from packit.config import PackageConfig` causes traceback:

~~~
Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/_pytest/config/__init__.py", line 372, in _getconftestmodules
    return self._path2confmods[path]
KeyError: local('/home/phracek/work/programming/packit/tests')

During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/_pytest/config/__init__.py", line 403, in _importconftest
    return self._conftestpath2mod[conftestpath]
KeyError: local('/home/phracek/work/programming/packit/tests/conftest.py')

During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/_pytest/config/__init__.py", line 409, in _importconftest
    mod = conftestpath.pyimport()
  File "/usr/lib/python3.7/site-packages/py/_path/local.py", line 668, in pyimport
    __import__(modname)
  File "/home/phracek/work/programming/packit/tests/__init__.py", line 1, in <module>
    from packit.utils import set_logging
  File "/home/phracek/work/programming/packit/packit/utils.py", line 15, in <module>
    from packit.config import PackageConfig
  File "/home/phracek/work/programming/packit/packit/config.py", line 17, in <module>
    from packit.utils import exclude_from_dict
ImportError: cannot import name 'exclude_from_dict' from 'packit.utils' (/home/phracek/work/programming/packit/packit/utils.py)
ERROR: could not load /home/phracek/work/programming/packit/tests/conftest.py
~~~

I have to move function `get_wildcard_resolved_sync_files` into packit/config.py file.
I guess, there are recursive calling imports.
But the function is closely related to config and I will move it.